src/main/java/FilterInput.java

```diff
@@ -0,0 +1,70 @@
+public class FilterInput {
+
+    /**
+     * Returns lateral location of the specified position.
+     *
+     * @param words String input by user.
+     */
```

Comment:
This comment describes the function of FilterInput class very well, is well organised and clear.